provider/clusterrosaclassic/cluster_rosa_classic_resource.go

```diff
@@ -237,24 +235,18 @@ func (r *ClusterRosaClassicResource) Schema(ctx context.Context, req resource.Sc
 			},
 			"replicas": schema.Int64Attribute{
 				Description: "Number of worker/compute nodes to provision. Single zone clusters need at least 2 nodes, " +
-					"multizone clusters need at least 3 nodes.",
+					"multizone clusters need at least 3 nodes. (only valid during cluster creation)",
 				Optional: true,
-				Computed: true,
 			},
 			"compute_machine_type": schema.StringAttribute{
 				Description: "Identifies the machine type used by the default/initial worker nodes, " +
 					"for example `m5.xlarge`. Use the `rhcs_machine_types` data " +
-					"source to find the possible values.",
+					"source to find the possible values. (only valid during cluster creation)",
 				Optional: true,
-				Computed: true,
-				PlanModifiers: []planmodifier.String{
```

Comment:
Why did you remove it? If the user updates it he will not get any error message and will think it was updated

Reply:
The CMT is just like the other fields that pertain to the default MP. They will be used to set the initial configuration for the worker MP, but any changes thereafter must be done via the MP resource.

The plan modifiers would cause the whole cluster to be deleted and recreated if the value is changed... I don't think that's what we want. To "change" the machine type, the pool must be recreated, not the cluster (I realize the worker pool can't actually be recreated, but I don't think destroying the cluster is a good solution).

Reply:
Can you test your binary code using importing cluster resource? 
According to planModifier - can you write a specific plan modifier for value cannot be changed?

Reply:
> Can you test your binary code using importing cluster resource?

Will do.

> According to planModifier - can you write a specific plan modifier for value cannot be changed?

I'm not sure if it's possible... I'll take a look and see if I can create one.

Reply:
Cluster import works.
This PR still need an approved tag